tests/qa_automation/execute_test_run.pm

```diff
@@ -0,0 +1,60 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary:  to execute testsuites in openQA from QA:Head by running its build up test run script
+# Maintainer: Yong Sun  <yosun@suse.com>
+
+use strict;
+use base "opensusebasetest";
+use base "upload_system_log";
+use base "junit_log";
+use utils;
+use testapi;
+
+sub run {
+    my $self = shift;
+
+    select_console('root-console');
+
+    my %args    = @_;
+    my $timeout = $args{timeout} || 180 * 60;
+    my $test = get_var("QA_TESTSUITE") . join(get_var("QA_VERSION"));
+    my $test_run = "test_$test-run";
+    my $run_log = "/tmp/$test-run.log";
+
+    #execute test run
+    script_run("/usr/share/qa/tools/$test_run |tee $run_log", $timeout);
```

Comment:
variable $test_run is not neccessary, since it only use in script_run

Reply:
this is need, $test_run is the running script name of each test suite, otherwise the test will not triggered :)